Newsletters/Yearn Finance Newsletter #42/Yearn Finance Newsletter #42.md

```diff
@@ -1,105 +1,105 @@
-# Yearn Finance Newsletter #42
+# Yearn Finance Bülteni #42
 
 ![](image1.png)
 
-Welcome to the 42nd edition of the Yearn Finance Newsletter. Our aim with this newsletter is to keep the Yearn and the wider crypto community informed of the latest news, including product launches, governance changes, and ecosystem updates. If youΓÇÖre interested in learning more about Yearn Finance, follow our official [Twitter](https://twitter.com/iearnfinance) and [Medium](https://medium.com/iearn) accounts.
+Yearn Finance Bülteninin 42. baskısına hoş geldiniz. Bu bültenle amacımız, Yearn'i ve daha geniş kripto topluluğunu ürün lansmanları, yönetim değişiklikleri ve ekosistem güncellemeleri de dahil olmak üzere en son gelişmelerden haberdar etmektir.Yearn Finance hakkında daha fazla bilgi edinmek istiyorsanız, resmi [Twitter](https://twitter.com/iearnfinance) ve [Medium](https://medium.com/iearn) hesaplarımızı takip edin.
 
-## **Summary**
+## **Özet**
 
 - The Yearn Blue Pill
-- Woofy Week 11 Snapshot
-- Migrate Or Die
-- New Curve Gauge Voting Strategy
-- Check Out The Pickle Jars
-- What It Takes To Be A Strategist
-- Audits Published For Yearn V2
-- Vaults at Yearn
-- Ecosystem News
+- Woofy 11. Hafta Anlık Görüntüsü
+- Migrate Or Die(ya Geçiş yap ya da Öl)
+- Yeni Bir Curve Ayarı İçin Oylama Stratejisi
+- Pickle Kavanozlarına Göz Atın
+- Stratejist Olmak İçin Gerekenler
+- Yearn V2 İçin Yayınlanan Denetimler
+- Yearn'deki Vault'la
+- Ekosistem Haberleri
 
-## **The Blue Pill NFTs**
+## **The Blue Pill NFT'leri**
 
-![](image2.png)
+Yearn'ın ilk yılını kutlamak için, size Yearn'ın geçmişi, bugünü ve geleceği için manevi bir rehber olan The Blue Pill'i getirmek için Project Galaxy ile bir kez daha işbirliği yaptık. Zemm tarafından yapılan The Blue Pill, en sadık 13.500 destekçimize sunulan güzel bir dizgi kitabıdır. Bu, Yearn topluluğunun çeşitli çiftçileri, paydaşları, seçmenleri ve bağışçıları için yedi benzersiz NFT dizisidir. YFI'nin hediyesini kutlamak için bize katılın!
 
-To celebrate the first year of Yearn, we have collaborated once again with Project Galaxy to bring you The Blue Pill, a spiritual guide for the past, present and future of Yearn. Made by Zemm, The Blue Pill is a beautifully-typeset book available to our 13,500 most loyal supporters. This is a series of seven unique NFTs for the various farmers, stakers, voters, and givers of the Yearn community. Join us in celebrating the gift of YFI!
+![](image2.png)
```

Comment:
Why was this reversed? Lets try to keep things the same as the original

Reply:
oops sorry copied and pasted in a wrong line.fixed

Reply:
Thanks